docs/core/howto/tap.rst

```diff
@@ -310,6 +310,107 @@ For a full list of cred plugins supported, see :api:`twisted.plugins <twisted.pl
 
 
 
+Deploy your application using python packages
+---------------------------------------------
+
+
+
+To deploy your application one possibility is to wrap it up in a python package.
+For this you need to write a special file ``setup.py``, which contains metadata
+of the package. You would have to extend the layout of your files like this:
+
+
+
+
+- **MyProject** - Top level directory
+
+
+- ``setup.py`` - Description file for the package
+
+
+  - **myproject** - Python package
+
+
+    - **__init__.py**
+
+
+  - **twisted**
+
+
+    - **plugins**
+
+      - ``myplugin.py`` - Plugin file
+
+
+
+
+
+.. code-block:: python
+
+
+    from distutils.core import setup
```

Comment:
I think `setuptools` is preferred by everyone now?  It provides nice things like `find_packages`, anyway, and no one really wants to maintain a manual list of packages.